tensorflow/core/BUILD

```diff
@@ -908,7 +908,7 @@ cc_library(
         ":lib_internal",
         ":proto_text",
         ":protos_all_cc",
-    ] + if_not_windows([":stream_executor"]),
+    ] + [":stream_executor"],
```

Comment:
No need to concatenate the two lists now: can you add `":stream_executor"` to the previous list?